include/lsst/jointcal/FittedStar.h

```diff
@@ -36,91 +36,85 @@ class FittedStar : public BaseStar, public PmBlock {
     friend class PhotometryFit2;
 
 private:
-    double mag;
-    double emag;
-    double col;
-    int gen;
-    double wmag;
-    unsigned indexInMatrix;
-    int measurementCount;
+    double _mag;
+    double _emag;
+    double _col;
+    int _gen;
+    double _wmag;
+    unsigned _indexInMatrix;
+    int _measurementCount;
     const RefStar* _refStar;
 
-    double flux2;
-    double fluxErr;
-    double fluxErr2;
+    double _flux2;
+    double _fluxErr;
+    double _fluxErr2;
 
 public:
     FittedStar()
             : BaseStar(),
-              mag(-1),
-              emag(-1),
-              col(0.),
-              gen(-1),
-              wmag(0),
-              indexInMatrix(-1),
-              measurementCount(0),
+              _mag(-1),
+              _emag(-1),
+              _col(0.),
+              _gen(-1),
+              _wmag(0),
+              _indexInMatrix(-1),
+              _measurementCount(0),
               _refStar(nullptr),
-              fluxErr(-1),
-              fluxErr2(-1) {}
-
-    FittedStar(const BaseStar& B)
-            : BaseStar(B),
-              mag(-1),
-              emag(-1),
-              col(0.),
-              gen(-1),
-              wmag(0),
-              indexInMatrix(0),
-              measurementCount(0),
+              _fluxErr(-1),
+              _fluxErr2(-1) {}
+
+    FittedStar(const BaseStar& baseStar)
+            : BaseStar(baseStar),
+              _mag(-1),
+              _emag(-1),
+              _col(0.),
+              _gen(-1),
+              _wmag(0),
+              _indexInMatrix(0),
+              _measurementCount(0),
               _refStar(nullptr),
-              flux2(-1),
-              fluxErr(-1),
-              fluxErr2(-1) {}
+              _flux2(-1),
+              _fluxErr(-1),
+              _fluxErr2(-1) {}
 
     //!
-    FittedStar(const MeasuredStar& M);
+    FittedStar(const MeasuredStar& measuredStar);
 
     //!
     void clearBeforeAssoc() {
-        indexInMatrix = -1;
-        measurementCount = 0;
+        _indexInMatrix = -1;
+        _measurementCount = 0;
         _refStar = nullptr;
-        wmag = 0;
+        _wmag = 0;
     }
 
     //!
     void dump(std::ostream& stream = std::cout) const {
         BaseStar::dump(stream);
-        stream << " mcount: " << measurementCount;
+        stream << " mcount: " << _measurementCount;
     }
 
     //!
-    int MeasurementCount() const { return measurementCount; }
-
-    //!
-    int& MeasurementCount() { return measurementCount; }
+    int getMeasurementCount() const { return _measurementCount; }
+    int& getMeasurementCount() { return _measurementCount; }
```

Comment:
Yes, it's used to increment and decrement measurementCount when constructing and cleaning the star lists.